go/pkg/sysdb/coordinator/table_catalog.go

```diff
@@ -367,6 +367,20 @@ func (tc *Catalog) GetCollections(ctx context.Context, collectionID types.Unique
 	return collections, nil
 }
 
+func (tc *Catalog) GetCollectionSize(ctx context.Context, collectionID types.UniqueID) (uint64, error) {
+	tracer := otel.Tracer
+	if tracer != nil {
+		_, span := tracer.Start(ctx, "Catalog.GetCollectionSize")
+		defer span.End()
+	}
+
+	total_records_post_compaction, err := tc.metaDomain.CollectionDb(ctx).GetCollectionSize(*types.FromUniqueID(collectionID))
```

Comment:
The `*types.FromUniqueID(collectionID)` part is a little strange. I guess it's fine if we're already doing it all over, but technically this could blow up if this returned nil.

Reply:
Updated